threatbus/src/connector.py

```diff
@@ -106,23 +106,54 @@ def _handle_threatbus_message(self, msg: str):
         if type(stix_msg) is Sighting:
             self._report_sighting(stix_msg)
         elif type(stix_msg) is Indicator:
-            self._ingest_indicator(stix_msg)
+            self._handle_indicator(stix_msg)
         else:
             self.opencti_helper.log_error(
                 f"Received STIX object with unexpected type from Threat Bus: {type(stix_msg)}"
             )
 
-    def _ingest_indicator(self, indicator: Indicator):
+    def _handle_indicator(self, indicator: Indicator):
         """
-        Ingests a STIX-2 Indicator into OpenCTI. Does nothing in case the
-        indicator already exists.
-        @param indicator The STIX-2 Indicator object to ingest
+        Handles a STIX-2 Indicator update received via Threat Bus. Does nothing
+        in case the indicator already exists or
```

Comment:
```suggestion
        in case the indicator already exists or ...
```

Something is missing...

Reply:
Fixed the comment.